docs/source/API/core/view/view_alloc.rst

```diff
@@ -44,8 +46,34 @@ Description
 
    ``args`` : Can only be a pointer to memory.
 
+
 .. cppkokkos:type:: ALLOC_PROP
 
    :cppkokkos:type:`ALLOC_PROP` is a special, unspellable implementation-defined type that is returned by :cppkokkos:func:`view_alloc`
    and :cppkokkos:func:`view_wrap`. It represents a bundle of allocator parameters, including the View label, the memory space instance,
    the execution space instance, whether to initialize the memory, whether to allow padding, and the raw pointer value (for wrapped unmanaged views).
+
+.. cppkokkos:type:: WithoutInitializing
+
+   :cppkokkos:type:`WithoutInitializing` is intended to be used in situation where default construction of `View` elements in its
+   associated execution space is not viable. This includes situations such as the construction of objects with virtual functions,
+   or `Views` of elements which do not have a default constructor. Typically, this option is used in conjunction with manual in-place `new`
+   construction of objects and manual destruction of elements.
+
+.. cppkokkos:type:: SequentialHostInit
+
+   :cppkokkos:type:`SequentialHostInit` is intended to be used to initialize elements that do not have a default constructor or destructor that
+   can be called inside a Kokkos parallel region. In particular this includes constructors and destructors which:
+
+   * allocate or deallocate memory
+   * create or destroy managed `Kokkos::View` objects
+   * call Kokkos parallel operations
+
+   When using this allocation option the `View` constructor/destructor will create/destroy elements in a serial loop on the Host.
+
+   .. warning::
+
+     `SequentialHostInit` can only be used when creating host accessible `View`s, such as `View`s with `HostSpace`, `SharedSpace`,
+     or `SharedHostPinnedSpace` as memory space.
```

Comment:
Should the `DualView` case be mentioned here too?

- https://github.com/kokkos/kokkos/pull/7456